pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -167,13 +184,13 @@ def on_epoch_end(self, epoch, logs=None):
         else:
             self.wait += 1
             if self.wait >= self.patience:
-                self.stopped_epoch = epoch
+                self.stopped_epoch = self.trainer.current_epoch
```

Comment:
all these class properties shall be listed in init, so it is clear what contains...

Reply:
Hm, it seems like all these properties are already listed in init.

Reply:
I don't see `stopped_epoch` in init...

Reply:
It is right here https://github.com/PyTorchLightning/pytorch-lightning/blob/1b101d82747269ccf82480a4f22c441b3e44f28e/pytorch_lightning/callbacks/pt_callbacks.py#L105